### PR TITLE
[docs-only] Update tus_upload.md

### PR DESCRIPTION
**Pull request by micbar**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by mmattel** [comment]
Hmmm,
I am not confident on this change because 'tus' is written in lowercase according to tus.io. we always respect the way of writing such names/terms and should not change this.

**Comment by micbar** [comment]
I know. But lowercase starting a sentence or headline is very weird IMO. In the block text i left it lowercase.

**Comment by mmattel** [comment]
@EParzefall and myself had some time ago an intense discussion exacly about that topic. How to deal with proper nouns, as this affects, as you have identified, the starting character at the beginning of a sentence and when we have sections where we use the headline capitalisation method. We both agreed that we consistently use the proper noun as it is written to respect the "owners" way of writing it - though, and I agree, it may look unusual.

**Comment by micbar** [comment]
@mmattel please review again, i did a workaround.

**Comment by mmattel** [comment]
What about:
* `Resumable Upload` --> `Resumable Upload (tus)`
* `Tus Official Site` --> `tus Official Site`

File: docs/apis/http/tus_upload.md [hunk 1]
```diff
@@ -8,11 +8,10 @@ geekdocFilePath: tus_upload.md
 geekdocCollapseSection: true
 ---
 
-Infinite Scale supports the tus resumable-upload protocol.
-tus is a robust, modular, and open protocol designed to resume large file uploads reliably over HTTP.
+Infinite Scale supports the tus resumable-upload protocol, which is a robust, modular, and open protocol designed to resume large file uploads reliably over HTTP.
 In situations where file uploads might be interrupted due to network issues, browser crashes, or other unforeseen interruptions,
 tus ensures that uploads can be resumed from the point of failure without losing data.
-This documentation shows some basic examples, refer [tus official site](https://tus.io/protocols/resumable-upload) for more details.
+This documentation shows some basic examples, refer [Tus Official Site](https://tus.io/protocols/resumable-upload) for more details.
```
Comment (EParzefall) [inline docs/apis/http/tus_upload.md:14]:
```suggestion
This documentation shows some basic examples, refer to the [tus official site](https://tus.io/protocols/resumable-upload) for more details.
```

Comment (EParzefall) [reply docs/apis/http/tus_upload.md:14]: No reason to capitalize "tus official site" here.

File: docs/apis/http/tus_upload.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "tus Upload"
+title: "Resumable Upload"
```
Comment (EParzefall) [inline docs/apis/http/tus_upload.md:2]:
```suggestion
title: "Resumable Upload (tus)"
```

File: docs/apis/http/tus_upload.md (hunk 1 above)
Comment (micbar) [reply docs/apis/http/tus_upload.md:14]: gnargh! looks ugly....

File: docs/apis/http/tus_upload.md (hunk 2 above)
Comment (micbar) [reply docs/apis/http/tus_upload.md:2]: no please not in headlines.

File: docs/apis/http/tus_upload.md (hunk 1 above)
Comment (micbar) [reply docs/apis/http/tus_upload.md]: changed.

**Review by mmattel — APPROVED** [review]

**Pull request merged**